src/js/components/Tab/doc.js

```diff
@@ -9,6 +9,9 @@ export const doc = (Tab) => {
     );
 
   DocumentedTab.propTypes = {
+    header: PropTypes.node.description(
+      'If specified, the entire panel header will be managed by the caller.'
```

Comment:
I'm not sure what 'panel' refers to in this context. I'm also wondering if `header` is the right term. I'm wondering if we should call it `label` instead?

Reply:
I'm following the pattern in Accordion. we call `header` there.

Reply:
I agree with the wording let me change it

Reply:
actually it makes sense your feedback. in accordion, it is actually a header. here it is not.